### ISSUE TITLE
Create wheels

### ISSUE THREAD

**Issue by tjstum** [simon-weber/gpsoauth#9]
Not a huge issue, but it seems like wheels are the way to go for distributing packages.

This one looks fairly easy (since there are no conditional dependencies, unlike gmusicapi). I was able to build a wheel (and install it for gmusicapi) without any fuss.

I'm not sure what your release procedure is, but it seems that all that's necessary is to just create a wheel when uploading. We can make a universal wheel (either with `--universal` given when running setup.py or making a setup.cfg file).

**Comment by simon-weber** [comment]
Nice; I didn't know it was so straightforward. I'll have to read up on wheels.

**Comment by thebigmunch** [comment]
Yeah, I have wheels released for my scripts and wrapper. See https://packaging.python.org/en/latest/distributing/#wheels
